index.js

```diff
@@ -27,24 +51,44 @@ function restartGame(){
     // Storing image sources for list of cards
     // Storing it as a list and not a matrix to make it a bit difficult to map list to the 3x4 grid
     const cardImageSrcs = [
-        'images/cards/inosuke.png',
-        'images/cards/nezuko.png',
+        //need upto 12 cards for level 1
         'images/cards/rengoku.png',
-        'images/cards/mask.png',
-        'images/cards/tanjiro.png',
+        'images/cards/rengoku.png',
+        'images/cards/zenitsu.png',
         'images/cards/zenitsu.png',
         'images/cards/inosuke.png',
+        'images/cards/inosuke.png',
         'images/cards/nezuko.png',
-        'images/cards/rengoku.png',
+        'images/cards/nezuko.png',
+        'images/cards/mask.png',
         'images/cards/mask.png',
         'images/cards/tanjiro.png',
-        'images/cards/zenitsu.png',
+        'images/cards/tanjiro.png',
+        //need upto 20 cards for level 2
+        'images/cards/inosuke2.png',
+        'images/cards/inosuke2.png',
+        'images/cards/kanao.png',
+        'images/cards/kanao.png',
+        'images/cards/kimetsu.png',
+        'images/cards/kimetsu.png',
+        'images/cards/nezuko2.png',
+        'images/cards/nezuko2.png',
+        //need upto 24 cards for level 3
+        'images/cards/tokito.png',
+        'images/cards/tokito.png',
+        'images/cards/genya.png',
+        'images/cards/genya.png',
     ];

```

Comment:
in this Array make the path appear only one time, and then where you are generating cardArray, generate this type of pattern there